### PR TITLE
Use new queries `listing` and `openOffer` instead of now deprecated `offer`

### PR DESCRIPTION
**Pull request by NicolasMahe**
### Description

Use new queries `listing` and `openOffer` instead of now deprecated `offer`

### Checklist

- [x] Update related changelogs

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -30,6 +30,6 @@
     "@graphql-codegen/typescript": "^1.23.0",
     "@graphql-codegen/typescript-graphql-request": "^4.5.5",
     "@graphql-codegen/typescript-operations": "^1.18.4",
-    "@nft/api-graphql": "^1.0.0-beta.46.2"
+    "@nft/api-graphql": "^1.0.0-beta.52-prerelease-8"
```
Comment (NicolasMahe) [inline packages/core/package.json]: Need to wait for non-prerelease version to merge.

File: packages/core/src/exchange/acceptOffer.ts [hunk 2]
```diff
@@ -57,8 +57,15 @@ export async function acceptOffer(
...
-  invariant(offer, "Can't find offer")
+  let offer = undefined
+  const { listing } = await sdk.FetchListing({ offerId })
+  if (listing) {
+    offer = listing
+  } else {
+    const { openOffer } = await sdk.FetchOpenOffer({ offerId })
+    invariant(openOffer, "Can't find offer")
+    offer = openOffer
+  }
```
Comment (antho1404) [inline packages/core/src/exchange/acceptOffer.ts]: I would move this to the calling functions that have the context and pass the offer as parameter. This `acceptOffer` is internal and called by the `acceptBid` and `buyToken` so the parent function have the required context

Comment (antho1404) [reply packages/core/src/exchange/acceptOffer.ts]: Done here [48eb35c](https://github.com/liteflow-labs/liteflow-js/pull/225/commits/48eb35cc594488751df1b76b32e199e16022c340)

**Review by antho1404 — APPROVED** [review]

File: packages/core/package.json (hunk 1 above)
Comment (antho1404) [reply packages/core/package.json]: I let you merge when the library is released

**Pull request merged**